rmw_zenoh_cpp/src/rmw_init.cpp

```diff
@@ -42,6 +42,10 @@ extern "C"
 // TODO(clalancette): Make this configurable, or get it from the configuration
 #define SHM_BUFFER_SIZE_MB 10
 
+// Environment variable for Zenoh logging
+const char * ZENOH_LOG_ENV_VAR = "RUST_LOG";
```

Comment:
My preference would be to move these definitions into `detail/zenoh_config.hpp` which is already included by `rmw_init.cpp` and `zenohd/main.cpp` below. Then you can define a `ZENOH_LOG_INFO_LEVEL_STR` and `ZENOH_LOG_DEBUG_LEVEL_STR` and use these in the `setenv` call.

Reply:
Done!